recipe-server/client/control_new/state/recipes/actions.js

```diff
@@ -48,29 +51,46 @@ export function fetchRecipe(pk) {
 }
 
 
-export function fetchAllRecipes() {
+export function fetchRecipesPage(pageNumber = 1) {
   return async dispatch => {
-    const requestId = 'fetch-all-recipes';
-    const recipes = await dispatch(makeApiRequest(requestId, 'v2/recipe/'));
+    const requestId = `fetch-recipes-page-${pageNumber}`;
+    const recipes = await dispatch(makeApiRequest(requestId, 'v2/recipe/', {
+      data: { page: pageNumber },
+    }));
 
-    recipes.forEach(recipe => {
+    recipes.results.forEach(recipe => {
       dispatch(recipeReceived(recipe));
     });
+
+    dispatch({
+      type: RECIPES_PAGE_RECEIVE,
+      pageNumber,
+      recipes,
+    });
   };
 }
 
 
-export function fetchFilteredRecipes(filters) {
+export function fetchFilteredRecipesPage(pageNumber = 1, filters = {}) {
   return async dispatch => {
     const filterIds = Object.keys(filters).map(key => `${key}-${filters[key]}`);
-    const requestId = `fetch-filtered-recipes-${filterIds.join('-')}`;
+    const requestId = `fetch-filtered-recipes-page-${pageNumber}-${filterIds.join('-')}`;
     const recipes = await dispatch(makeApiRequest(requestId, 'v2/recipe/', {
```

Comment:
Try/catch thing again. Do we have an error notification/toast system set up? Maybe that needs to be another PR all together.